### PR TITLE
sdl: new package at 1.2.15

### PR DESCRIPTION
**Pull request by alaughlin**
## Description of package
https://www.libsdl.org/download-1.2.php

## Checklist

- [x] Latest upstream version.
- [x] I agree to maintain this package (**required**).
- [x] I agree to notify KISS if I can no longer maintain this package (**required**).

### CODE REVIEW

File: community/sdl/build [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/sh -e
+
+sed -e '/_XData32/s:register long:register _Xconst long:' \
```
Comment (dylanaraps) [inline community/sdl/build]: What does this do?

Comment (alaughlin) [reply community/sdl/build]: fixes a bug that prevents it from compiling with x11 support, I added a comment to the build file

here's some relevant info:
https://www.linuxquestions.org/questions/linux-from-scratch-13/can%27t-compile-sdl-4175469154/

The above thread includes a link to a patch, but LFS has the simpler solution I ended up using:
http://www.linuxfromscratch.org/blfs/view/svn/multimedia/sdl.html

I figured a simple `sed` was better than a patch file?

Comment (dylanaraps) [reply community/sdl/build]: Sounds good. :+1:

Was just wondering. :)

Comment (alaughlin) [reply community/sdl/build]: :)

File: community/sdl/build [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#!/bin/sh -e
+
+# fixes bug that prevents SDL from building with X11 support
```
Comment (dylanaraps) [inline community/sdl/build]:
```
# Fixes bug that prevents SDL from building with X11 support.
```

Comment (alaughlin) [reply community/sdl/build]: fixed 😆

**Comment by dylanaraps** [comment]
Thanks :)

**Pull request merged**